torchgeo/trainers/byol.py

```diff
@@ -339,15 +334,13 @@ def config_task(self) -> None:
         # initialize the weights from new channel with the red channel weights
         copy_weights = 0
         # Copying the weights from the old to the new layer
-        new_layer.weight[:, : layer.in_channels, :, :].data[
-            ...  # type: ignore[index]
-        ] = Variable(layer.weight.clone(), requires_grad=True)
+        new_layer.weight[:, : layer.in_channels, :, :].data[:] = Variable(
+            layer.weight.clone(), requires_grad=True
+        )
         # Copying the weights of the old layer to the extra channels
         for i in range(in_channels - layer.in_channels):
             channel = layer.in_channels + i
-            new_layer.weight[:, channel : channel + 1, :, :].data[
-                ...  # type: ignore[index]
```

Comment:
mypy doesn't like `...` for some reason, `:` should be equivalent as far as I know.

Reply:
I think this is okay here